### PR TITLE
readme added

### PR DESCRIPTION
**Pull request by NW91**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +59,44 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+Follow this Heroku link for the deployment of this Project 
+
+## Built With
+
+* [Heroku](https://devcenter.heroku.com/articles/github-integration) - The web framework used
+* [NodeJs](https://nodejs.org/en/)
+* [MongoDB](https://github.com/mongodb/docs)
+* [Express](https://expressjs.com/)
+* [React](https://github.com/reactjs/reactjs.org)
+* [Passport](https://github.com/jaredhanson/passport-github)
```
Comment (CrypticWoodWhite) [inline README.md:73]: this should be https://github.com/jaredhanson/passport

File: README.md [hunk 2]
```diff
@@ -37,8 +59,44 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+## Versioning
+
+We use [GitHub](http://github.com/) for versioning. For the versions available, see the [tags on this repository](https://github.com/center-aisle/middle-ground). 
+
+## Authors
+
+* [Naomi Woodruff](https://github.com/naywood)
+    SDE: Designed and engineered the frontend by making the pages, creating user examples, and aesthetic framework for this app using materialize.
+* [Bryan Kelley](https://github.com/bryanrkelley)
+    SDE: Designed and gngineered the frontend by making the pages, setting up the components, creating the aethetic framework for this app using materialize, and setting up the continuous integration with Heroku.
```
Comment (CrypticWoodWhite) [inline README.md:90]: mispelled engineered

Comment (CrypticWoodWhite) [reply README.md:90]: and aesthetic

Comment (CrypticWoodWhite) [reply README.md:90]: Oh, and keep all your tenses consistent - designed, engineered, set up, created, set up

Comment (CrypticWoodWhite) [reply README.md:90]: that last comment goes for the entire readme

**Review by naywood — APPROVED** [review]
README

**Pull request merged**